SpywareFilter/sections/tracking_servers_firstparty.txt

```diff
@@ -26,6 +26,8 @@
 ! Disabled in DNS filter
 ||telemetry.mozilla.org^
 !
+||statclick.nate.com^
+||statetc.nate.com^
```

Comment:
Have you seen it on desktop?

Reply:
Yes. `statetc.nate.com` domain is also used on desktop version of the website.
However, I cannot see that `statclick.nate.com` domain  is also used on there. I will move to `mobile.txt`.